banner-plugin/src/admin/routes/banners/page.tsx

```diff
@@ -1,15 +1,173 @@
+      </FocusModal.Trigger>
+      <FocusModal.Content>
+        <FormProvider {...form}>
+          <form onSubmit={form.handleSubmit(handleSubmit)} className="flex h-full flex-col overflow-hidden">
+            <FocusModal.Header>
+              <div className="flex items-center justify-end gap-x-2">
+                <FocusModal.Close asChild>
+                  <Button size="small" variant="secondary">
+                    Cancel
+                  </Button>
+                </FocusModal.Close>
+                <Button type="submit" size="small">
+                  Save
+                </Button>
+              </div>
+            </FocusModal.Header>
+            <FocusModal.Body>
+              <div className="flex flex-1 flex-col items-center overflow-y-auto">
+                <div className="mx-auto flex w-full max-w-[720px] flex-col gap-y-8 px-2 py-16">
+                  <div>
+                    <Heading className="capitalize">Create Banner</Heading>
+                  </div>
+                  <div className="grid grid-cols-2 gap-4">
+                    <Controller
+                      control={form.control}
+                      name="title"
+                      render={({ field }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Title
+                            </Label>
+                          </div>
+                          <Input {...field} placeholder="Enter banner title" />
+                        </div>
+                      )}
+                    />
+
+                    <Controller
+                      control={form.control}
+                      name="link"
+                      render={({ field }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Link
+                            </Label>
+                          </div>
+                          <Input {...field} placeholder="Enter banner link" />
+                        </div>
+                      )}
+                    />
+
+                    <Controller
+                      control={form.control}
+                      name="image"
+                      render={({ field }) => (
+                        <div className="flex flex-col space-y-2">
+                          <div className="flex items-center gap-x-1">
+                            <Label size="small" weight="plus">
+                              Image
+                            </Label>
+                          </div>
+                          <Input {...field} placeholder="Enter image URL" />
+                        </div>
+                      )}
+                    />
```

Comment:
I was wondering if we could use this image input instead (similar to the one found when creating a product)

<img width="760" alt="Screenshot 2024-12-30 at 20 07 34" src="https://github.com/user-attachments/assets/9f65ff6a-e244-411b-8ea7-4b52d8cf809d" />

because there will be 2 endpoints:

1. upload the images (it will return the url/image id)
2. create the banner (passing the image id from above)